.github/workflows/build-electron-dev.yml

```diff
@@ -1,7 +1,4 @@
-on:
-  push:
-    branches:
-      - dev
+on: push
```

Comment:
In gitlab you could define "branch dev + for any PR" then you don't have'm running on branches without PR. But yeah if computation load doesn't matter... just go ahead :)

Reply:
Since we don't pay for CI (well, there's not free lunch etc etc, but we don't transfer cash at least) I thought it could actually be beneficial to see if CI passes even on WIP branches